internal/cli/atlas/cloudproviders/accessroles/list.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package accessroles
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type ListOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store store.CloudProviderAccessRoleLister
+}
+
+func (opts *ListOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var listTemplate = `ID	PROVIDER{{range .AWSIAMRoles}}
+{{.RoleID}}	{{.ProviderName}}{{end}}
```

Comment:
[nit] provider is always AWS, can we please add ARN and external ID as well here?

Reply:
I thought that the reason why this command is not inside the AWS directory is that, in the future, the list command would show different providers as well. Am I wrong? If I am wrong, why aren’t we moving this command inside the aws folder

Reply:
sorry, my comment is not about removing the provider but that only listing ID and provider name is not helpful, as they are all AWS, users may need more info like ARNs to know the difference